js/kucoin2.js

```diff
@@ -1081,8 +1080,11 @@ module.exports = class kucoin2 extends Exchange {
         //
         let errorCode = this.safeString (response, 'code');
         if (errorCode in this.exceptions) {
+            const message = this.safeString (response, 'msg', '');
```

Comment:
Pardon, why would you remove it from exceptions? I suggest that we do check errorCode in this.exceptions, and if it's not there, check `const message = this.safeString ('response, 'msg') in this.exceptions. Can you rewrite it properly and restore the exception back in the options?

Reply:
Sounds good. Other exchanges used the message checking in `handleErrors` so I assumed it was the way to go.